README.md

```diff
@@ -1,184 +1,30 @@
-# Python Discord Code Jam Repository Template
```

Comment:
Hi @Vinyzu , Good work. 

it's LGTM, just that I think we need to improve the README afterwards.

Reply:
Yes, i would describe the ReadMe as an active WIP. Will push more changes over time and improve it.